### DISCUSSION TITLE
Albums being split with multiple album artists

### DISCUSSION THREAD

**Discussion by TheMarstonConnell** [gerbera/gerbera#3057] [Q&A] [answered]
I was wondering if there was any easy way to prioritize grouping songs by parent folder? I have an album that has one song with a feature that gets split into a new album and the AlbumArtist tag is the featured artist instead of the actual album artist.

The featured track being split into its own album:
<img width="1141" alt="image" src="https://github.com/gerbera/gerbera/assets/34043723/49d069d7-48ef-4634-b1e5-5b6bc80cd5c4">

The actual album:
<img width="1143" alt="image" src="https://github.com/gerbera/gerbera/assets/34043723/ae936c83-5d6f-4aa4-b3fd-ab894a5a02ff">

Is there some import setting I can change? 

Thanks

**Comment by KarlStraussberger** [comment, marked as answer]
The solution is to set the AlbumArtist to the one artist (Wilco) that released the album. You still can have a separate artist for each track.
